src/Traits/SoftekGetterTrait.php

```diff
@@ -0,0 +1,54 @@
+<?php
+declare(strict_types=1);
+
+namespace DexproSolutionsGmbh\SoftekBarcodeWrapper\Traits;
+
+use FFI\CData;
+
+/**
+ * This Trait can be implemented in each business logic class like transaction classes, service classes
+ * or controller classes. Be sure that the global properties instance and ffi are declared and usable.
```

Comment:
I think we should talk about the terms `business logic`, `transaction classes` and `service classes`.

This package implements a wrapper arround a FFI. That is a rather technical topic. In this context I really don't know what a transaction class or service class is supposed to be. Of course, those terms are often used when implementing a web app / api. But this package has nothing to do with that. IMO `SoftekBarcodeAPI` isn't a service class or anything similar. In the context of this package, these terms really have no meaning to me.

I think, here less is more: "This trait implements getter / setter methods on the ffi object wrapped by this library."

Also, traits are not *implemented* by classes, they are *used*.

Reply:
I have adjusted the comments